### PR TITLE
0.7.0 release

### PR DESCRIPTION
**Pull request by gonuke**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Update recipe for official v0.7.0 release

### CODE REVIEW

**Comment by gonuke** [comment]
@conda-forge-admin, please rerender

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -1,7 +1,10 @@
...
-
+  
 enable_openmc:
-  - noopenmc
-  - openmc
+    - noopenmc
+    - openmc
+    
+channel_targets:
+  - conda-forge pyne_rc
```
Comment (bam241) [inline recipe/conda_build_config.yaml:10]: is the channel still correct there ?

**Comment by gonuke** [comment]
@conda-forge-admin, please rerender

**Comment by bam241** [comment]
LGTM!
thx a lot @gonuke for handling this.

Merging !

**Pull request merged**